openreview/conference/builder.py

```diff
@@ -714,13 +714,13 @@ def set_authors(self):
         authors_group = self.__create_group(self.get_authors_id(), self.id, public=True)
         return self.webfield_builder.set_author_page(self, authors_group)
 
-    def setup_matching(self, is_area_chair=False, affinity_score_file=None, tpms_score_file=None, elmo_score_file=None, build_conflicts=False):
+    def setup_matching(self, is_area_chair=False, affinity_score_file=None, tpms_score_file=None, elmo_score_file=None, build_conflicts=False, delete_existing_conflicts=True):
```

Comment:
why do you want to delete the conflicts and don't create them again?

build_conflicts = True does that. If you pass build_conflicts = False the conflicts shouldn't be deleted/created.

Reply:
Agreed, change has been reverted.